envConfig/live.env

```diff
@@ -6,6 +6,10 @@ SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
 SIMORGH_CONFIG_URL=https://config.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.bbc.com/ws/includes
 SIMORGH_INCLUDES_BASE_AMP_URL=https://news.files.bbci.co.uk
+SIMORGH_ADS_SCRIPT_TEST=https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js
```

Comment:
Since these paths don't seem to differ per live.env/test.env/local.env, could they not be just constants within the Ads container code?

Reply:
I was thinking initially that we could have a base URL defined in config, which in this case would be `https://gn-web-assets.api.bbc.com/ngas` for live and `https://gn-web-assets.api.bbc.com/ngas/test/` for test. I don't think that would work with the requirement to be able to override with a query parameter though, so moving these back into the ad container's probably a better bet.

Reply:
@mmcculloch are we deciding to keep these in the config or move them to the Ads container?

Reply:
I will move them into ads container :)